outline/intro.md

```diff
@@ -195,7 +195,7 @@ Introduction to Programming with Clojure
 
 **R**ead, **E**val, **P**rint, **L**oop
 
-![Instarepl](img/instarepl.png)
+![Nightcode's repl](img/repl.png)
```

Comment:
@davidchambers before merging pull request, would you give those a try? I don't think there's problem since Berlin team has already used. If all works, go ahead and merge the pull request.

For some reason, on my laptop (El Capitan), key combination didn't work. I may have missed some, though.